content/use-cases/open-a-payment-channel-to-enable-an-inter-exchange-network.md

```diff
@@ -0,0 +1,84 @@
+# Open a Payment Channel to Enable an Inter-Exchange Network
+
+As a digital asset exchange, do you make multiple XRP payments to another exchange with some degree of frequency? If this usage pattern sounds familiar, consider opening an XRP Ledger [payment channel](payment-channels.html) between your exchange (the _payer_ exchange) and the other exchange (the _payee_ exchange). A payment channel enables you to send one-way, "asynchronous" XRP payments that can be divided into very small increments and settled later. Here are some of the benefits of using a payment channel to send XRP instead of using individual payment transactions:
+
+- **Achieve faster processing:** A standard payment transaction involves submitting an XRP Ledger transaction and waiting for a new ledger version that includes the transaction to be approved by [consensus](https://developers.ripple.com/consensus.html). When you use a payment channel to send XRP, creation and verification of a claim, which guarantees the payment of XRP, all happen outside of the consensus process. This means that the payer exchange can guarantee XRP payments to the payee exchange at a rate limited only by the participants' ability to create and verify the digital signatures of the claims.
+
+- **Lower transaction costs:** The payee exchange can choose to take the claims it has received from the payer exchange and redeem them in batches to receive the guaranteed XRP amounts. Redeeming a batch of claims in a single transaction incurs a _single_ set of transaction costs (any applicable reference transaction, load, and open ledger costs), while processing multiple XRP payment transactions (one per claim without a payment channel in place, for example) to receive the same amount of XRP incurs _multiple_ sets of transaction costs. ***TODO: accurately stated?***
```

Comment:
Thank you for these comments! 🙏  Just to hep me understand the misuse here 😂 -- I called them reference transaction, load, and open ledger costs bc of the way transaction costs are documented here: https://developers.ripple.com/transaction-cost.html. Should I call them "reference transaction, load, and open ledger potential minimums"? Or is there no need to call them out? Or are we talking about different things? I want to use the correct language -- but I think I'm missing something.

For "XRP payment transactions" -- thank you for this clarification. Just to be sure I understand - the comparison I intended to make is this: Instead of making multiple Payments, for which you'd be charged costs for _multiple_ transactions -- you can redeem a batch of claims in a single transaction (that ends up having the same effect as the multiple Payments, for example), incurring the cost of a _single_ transaction. Is this an incorrect description of why/how the ability to redeem payment channel claims in batch can lower costs? Missing something again - thank you for your patience!

Reply:
There's only one transaction cost. You don't incur a "set of transaction costs".

It's just that how much you have to pay for the transaction cost depends on several different factors. The transaction cost page is a little confusing, and for that I'm sorry. It's not an easy concept to explain, and it's evolved to become more confusing since that page was originally written. 

Basically, you're going to pay one transaction cost, probably whichever is highest among the costs mentioned. Except not always, because you don't have to pay the open ledger cost if you don't mind waiting in the queue. And the some transactions have to pay more than the "reference transaction cost" because that's just the cost of a so-called "reference transaction" that doesn't have an alternate minimum cost because of things like multi-sigs and crypto-condition fulfillments.

Reply:
Also, after re-reading, I think the second part makes sense. Maybe it's a little densely worded. But yeah, you seem to have the correct understanding of the idea of individual payments vs. batch claims.